docs/adding-new-models.md

```diff
@@ -183,7 +183,8 @@ A minimal bidirectional end-to-end check:
 from megatron.bridge import AutoBridge
 
 # HF → Megatron
-bridge = AutoBridge.from_hf_pretrained("<org>/<model-id>", trust_remote_code=True)
+# Set trust_remote_code to True only if you trust the repository
```

Comment:
this doc needs to be added for updating the allow list when adding a new model to the bridge